pkgs/development/python-modules/empyrical/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, numpy
+, pandas
+, scipy
+, pandas-datareader
+, parameterized
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "empyrical";
+  version = "0.5.5";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "quantopian";
+    repo = pname;
+    rev = version;
+    sha256 = "1cjplgh7kqqkav1l42r91h1s575xjv68s8awxyf5749wg1jwkfja";
+  };
+
+  # one test is known to fail: https://github.com/quantopian/empyrical/issues/131
+  patchPhase = ''
+    sed -i 's/    def test_perf_attrib_simple(self):/    @unittest.skip("known to fail")\n    def test_perf_attrib_simple(self):/' empyrical/tests/test_perf_attrib.py
+  '';
+  checkInputs = [ parameterized ];
+  checkPhase = ''
+    ${python.interpreter} runtests.py
+  '';
+
+  propagatedBuildInputs = [ numpy pandas scipy pandas-datareader ];
```

Comment:
Normally this is put before the checkPhase because the build phase is executed before the checkPhase.

Reply:
Thanks, this is updated